thavalon-server/src/connections/account_handlers.rs

```diff
@@ -104,14 +123,16 @@ pub async fn handle_add_user(
         }
     };
 
-    let mut db_user: DatabaseAccount = new_user.into();
-    db_user.hash = hash;
-    if let Err(e) = database::create_new_user(&db_user).await {
-        log::info!("{:?}", e);
-        return Err(reject::custom(DuplicateAccountRejection));
-    }
-    log::info!("Successfully added {} to the database.", db_user.email);
-    let (jwt, refresh_token) = token_manager.create_jwt(&db_user.email).await;
+    let player_id =
+        match database::create_new_user(&new_user.email, &hash, &new_user.display_name).await {
+            Ok(id) => id,
+            Err(e) => {
+                log::info!("{:?}", e);
```

Comment:
I thought about this more and started switching them to warning, but I really started to dislike it. Following this pattern, we'd be log::warning on invalid passwords, non-existent users, and other cases that are likely to occur.

For actual errors, the thing throwing the error already logs and error. Case and point, if an AccountError::UnknownError occurs, which shouldn't happen, we log it as an error at the DB level, so it would still appear in the logs. If you disagree, we can have Ben tiebreak.